vms/registration/templates/registration/signup_volunteer.html

```diff
@@ -1,8 +1,28 @@
 {% extends "vms/base.html" %}
 
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+   <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script> 
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
 
 {% block content %}
 
+<script>
```

Comment:
@sarayd  Can you please separate the JS code. Just look if this can be put into some already existing JS file or add a new JS file and put this into that, this will make things more clean and clear.